core-api/src/test/java/com/optimizely/ab/OptimizelyTest.java

```diff
@@ -3383,9 +3382,181 @@ public void getFeatureVariableValueReturnsVariationValueWhenUserGetsBucketedToVa
             FeatureVariable.VariableType.STRING
         );
 
+        logbackVerifier.expectMessage(
+            Level.INFO,
+            "Feature \"" + validFeatureKey + "\" for variation \"Gred\" was not enabled. " +
+                "The default value \""+ expectedValue +"\" for \""+ validVariableKey +"\" is being returned."
+        );
+
         assertEquals(expectedValue, value);
     }
 
+    /**
+     * Verify that the {@link Optimizely#getFeatureVariableString(String, String, String, Map)}
+     * notification listener of getFeatureVariableString is called when feature is in experiment and feature is true
```

Comment:
What does notification listener have to do with this change? Please fix docstring.